### PR TITLE
Fixes for deprecated functions and mainline uboot

### PR DESCRIPTION
**Pull request by panicking**
Couple of changes for uuu tools

### CODE REVIEW

File: uuu/fat_write.lst [hunk 1]
```diff
@@ -7,5 +7,6 @@ uuu_version 1.1.4
 
 FB: ucmd setenv fastboot_buffer ${loadaddr}
 FB: download -f _image
+FB: ucmd if test -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize;
```
Comment (nxpfrankli) [inline uuu/fat_write.lst:10]: where set $filesize environment?

Comment (nxpfrankli) [reply uuu/fat_write.lst:10]: I found it. latest uboot fastboot set filesize

Comment (panicking) [reply uuu/fat_write.lst:10]: @nxpfrankli sorry I make a mistake on script. I need to fix it out

Comment (panicking) [reply uuu/fat_write.lst:10]: ucmd if test -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize; fi

Comment (panicking) [reply uuu/fat_write.lst:10]: FB: ucmd if test ! -n "$fastboot_bytes"; then setenv fastboot_bytes $filesize; fi

**Pull request merged**